obp-api/src/main/scala/code/util/Helper.scala

```diff
@@ -239,9 +239,9 @@ object Helper{
    /**
     * Used for version extraction from props string
     */
-  val matchKafkaVersion = "kafka_v([0-9a-zA-Z_]+)".r
-  val matchAnyKafka = "^kafka.*$".r
-  val matchAnyStoredProcedure = "^stored_procedure.*$".r
+  val matchKafkaVersion = "kafka_v([0-9a-zA-Z_]+)|^star$".r
+  val matchAnyKafka = "^(kafka.*|star)$|^star$".r
```

Comment:
matchAnyKafka  should be `"^(kafka.*|star)$".r`
`starConnector_supported_types` also need to be check, whether it contains "kafka".
So, please check the follow code:
`case Helper.matchAnyKafka() if ( APIUtil.getPropsAsBoolValue("kafka.user.authentication", false) &&`